extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -116,5 +125,23 @@ describes.realWin(
         ).to.deep.equal(tagData);
       });
     });
+
+    it('should call analytics service on attachment state update', async () => {
+      const trigger = env.sandbox.stub(analytics, 'triggerEvent');
+      await setupShoppingTagAndData();
+      await shoppingTag.shoppingTagEl_.click();
+      expect(trigger).to.have.been.calledWith(
+        StoryAnalyticsEvent.SHOPPING_PDP_VIEW
+      );
+    });
```

Comment:
Shopping tag clicks should not be what triggers `SHOPPING_PDP_VIEW`. This test can be removed.